scripts/rollup/results.json

```diff
@@ -1,17 +1,17 @@
 {
-  "branch": "master",
+  "branch": "warnForKeysInFragmentsRefactor",
```

Comment:
Won't we get terrible merge conflicts soon once everyone has a different base here? :)